tests/iam/certhandler/mocks/privatekey.hpp

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_IAM_MOCKS_PRIVATEKEY_HPP_
+#define AOS_IAM_MOCKS_PRIVATEKEY_HPP_
+
+#include "aos/common/crypto.hpp"
+
+namespace aos {
+namespace crypto {
+
+/**
+ * RSA private key.
+ */
+class RSAPrivateKeyStub : public PrivateKeyItf {
```

Comment:
Why do you need different KeyTypes? I guess one `PrivateKeyMock` would be enough, wouldn't be?